package.json

```diff
@@ -15,9 +15,9 @@
     "@hookform/resolvers": "^3.0.1",
     "@react-oauth/google": "^0.10.0",
     "@reduxjs/toolkit": "^1.9.3",
-    "@testing-library/react": "14.0.0",
```

Comment:
are you sure?

Reply:
if you mean deleting  "@testing-library/react":, its because is a devDependecy, not a dep, i moved to dev dependecies